addons/libs/images.lua

```diff
@@ -0,0 +1,344 @@
+--[[
+    A library to facilitate image primitive creation and manipulation.
+]]
+
+local images = {}
+local meta = {}
+
+windower.prim.saved_images = {}
+local dragged
```

Comment:
Okay I'll look into putting it back. I personally hated always missclicking the screen, and dragging the Text fields on the pointwatch addon, but it is a lot better easier than positioning with POS commands.